templates/repo/view_list.tmpl

```diff
@@ -34,7 +34,7 @@
 					</span>
 				{{end}}
 			</th>
-			<th class="text grey right age">{{if .LatestCommit}}{{if .LatestCommit.Committer}}{{TimeSince .LatestCommit.Committer.When $.locale}}{{end}}{{end}}</th>
+			<th class="text grey right age">{{if .LatestCommit}}{{if .LatestCommit.Committer}}{{TimeSince .LatestCommit.Committer.When $.locale "tense=\"past\""}}{{end}}{{end}}</th>
```

Comment:
Can the escaping be avoided or maybe can we somehow pass a dict-like?

Reply:
Avoided escaping in https://github.com/go-gitea/gitea/pull/24106/commits/a54bcb078b0f53ef2be8ffb82f06f97e5c003405